jwst/regtest/test_miri_image.py

```diff
@@ -68,7 +68,7 @@ def run_image3(run_image2, rtdata_module):
         "--steps.tweakreg.snr_threshold=200",
         "--steps.tweakreg.use2dhist=False",
         "--steps.tweakreg.minobj=4",
-        "--steps.source_catalog.snr_threshold=20",
+        "--steps.source_catalog.snr_threshold=10",
```

Comment:
This change is to get back to detecting the 5 sources in the MIRI Image regtest.  One source got overlooked when the background subtraction bug was fixed.  See #4858.

Reply:
After this is merged, we'll have to OKify the source catalog test in `regtests/test_miri_image.py`.